blocks/card/card.js

```diff
@@ -1,36 +1,27 @@
-import { createOptimizedPicture } from '../../scripts/aem.js';
+import { cleanText } from '../../blocks-helpers/textHelpers.js';
 
 export default function decorate(block) {
+  const cardData = {
+    img: block.children[0].firstElementChild.firstElementChild,
+    textContent: block.children[1].children[0].children,
+    url: block.children[2].textContent.trim()
+  };
+
   const cardContainer = document.createElement('div');
   cardContainer.className = 'card-container';
   
-  const card = document.createElement('div');
+  const card = document.createElement('a');
   card.className = 'card';
 
-  const row = block.firstElementChild;
-  if (row) {
-    const [cardImage, cardBody] = row.children;
-
-    if (cardImage) {
-      const img = cardImage.querySelector('img');
-      if (img) {
-        img.className = 'card__image';
-        card.append(img);
-      }
-    }
-
-    if (cardBody) {
-      cardBody.className = 'card__body';
-      const [titleElement, descriptionElement] = cardBody.children;
-      if (titleElement) titleElement.className = 'card__title util-title-s';
-      if (descriptionElement) descriptionElement.className = 'card__description util-body-xs';
-      card.append(cardBody);
-    }
-  }
+  cardData.img.classList.add('card__image');
```

Comment:
We need to add an alt text attribute, whether it has content or not. I believe these images are decorative in all circumstances, but I am not for certain. 

[For a screenreader to know an image is decorative, add empty alt text.](https://www.w3.org/WAI/tutorials/images/decorative/) We have to have attribute, rather than just leaving it out, so the screen reader gets the signal to ignore it.

```suggestion
cardData.img.classList.add('card__image');
cardData.img.setAttribute('alt', '');
```